alembic/versions/bc442d63d7d3_move_forward_current_value_for_heron.py

```diff
@@ -0,0 +1,26 @@
+"""Move forward current value for heron
+
+Revision ID: bc442d63d7d3
+Revises: a32c725ae353
+Create Date: 2021-10-13 14:07:48.425000
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "bc442d63d7d3"
+down_revision = "a32c725ae353"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Last value in database is 2564197, we want to move forward past this value
+    op.execute("ALTER SEQUENCE heron RESTART WITH 2564300;")
+
+
+def downgrade():
+    # Actual value in Baracoda is 2156143, but if we rollback we can leave a bit of space
+    # Please retrieve the last actual value and edit if you plan to downgrade
```

Comment:
Just a suggestion, but as this requires intervention by a developer and this comment could go unnoticed, you could put an expectation in here that an environment variable be set to allow the downgrade to progress and otherwise raise an exception.  If the developer understands what they're doing, they have to set the environment variable to acknowledge they know that just downgrading could be bad if they don't know what the upgrade is going to do afterwards.

Reply:
Yes, this downgrade procedure seems dangerous as it can give the impression that it downgraded safely while it
didnt do anything instead; I'll replace it with a more descriptive exception.